packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring.ts

```diff
@@ -212,30 +212,11 @@ export class StackdriverStatsExporter implements StatsEventListener {
       count: distribution.count.toString(),
       mean: distribution.mean,
       sumOfSquaredDeviation: distribution.sumSquaredDeviations,
-      bucketOptions: {
-        explicitBuckets:
-            {bounds: this.getBucketBoundaries(distribution.buckets)}
-      },
-      bucketCounts: this.getBucketCounts(distribution.buckets)
+      bucketOptions: {explicitBuckets: {bounds: [0, ...distribution.buckets]}},
+      bucketCounts: distribution.bucketCounts
```

Comment:
(Correct me if I am wrong) I think you forgot to add 0 here, please add tests...